openid-4-verifiable-presentations-1_0.md

```diff
@@ -136,6 +136,11 @@ Implementations can also be built on top of OpenID Connect Core, which is also b
 
 Any of the OAuth 2.0 related specifications, such as [@RFC9126] and [@RFC9101], and Best Current Practice (BCP) documents, such as [@RFC8252] and [@I-D.ietf-oauth-security-topics], can be implemented on top of this specification.
 
+In summary, OpenID for Verifiable Presentations is a framework that requires profiling
+to achieve interoperability. Out of the
```

Comment:
This text is quite generic and I'm not sure if I would agree with the phrasing. The issue I have with the phrasing is that a profile can also have optional parameters, not only mandatory ones.

IMO, a profile does the following for a specific use case or ecosystem:
- define choices of values for mandatory and optional features, e.g., allowed credential format identifiers.
- define what optional features do not apply or mandate optional features
- optionally extend OID4VP if new features are required

This description is still vague but I could probably live with this better.

However, I think it would be better to define concretely what exactly has to be further defined by a profile.